package.json

```diff
@@ -49,5 +49,8 @@
   "lint-staged": {
     "*.{js,ts,sol}": "npx prettier --write",
     "*.cairo": "cairo-format -i"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.0"
```

Comment:
we don't use `devDepencencies` since the dev/prod relationship in not handled the same way for crypto. let's move it to `dependencies` instead